game/Assets/Scripts/Game/StateMachine/GameStateMachine.cs

```diff
@@ -11,16 +13,49 @@ public class GameStateMachine : StateMachine<GameStates>
     public UpgradeMenu UpgradeMenu;
     public SentryPlacingMenu SentryPlacingMenu;
     public GameOverMenu GameOverMenu;
+
+    class DisposableSpan : IDisposable
+    {
+        private ISpan _span;
+        
+        public DisposableSpan(ISpan span) => _span = span;
+
+        public void Dispose() => _span?.Finish(SpanStatus.Ok);
+    }
+
+    private ITransaction _startTransaction;
     
+    public void Awake()
+    {
+        _startTransaction = SentrySdk.StartTransaction("initialize", "State Machine");
+        SentrySdk.ConfigureScope(scope => scope.Transaction = _startTransaction);
+        // The transaction will be closed by BugSpawner.
+        base.Awake();
+    }
     protected override void Initialize()
     {
         base.Initialize();
         
-        _states.Add(GameStates.StartMenu, new GameStateStart(this));
-        _states.Add(GameStates.Upgrading, new GameStateUpgrading(this));
-        _states.Add(GameStates.SentryPlacing, new GameStatePlacing(this));
-        _states.Add(GameStates.Fighting, new GameStateFighting(this));
-        _states.Add(GameStates.GameOver, new GameStateGameOver(this));
+        using (new DisposableSpan(_startTransaction.StartChild("StartMenu", "state")))
+        {
+            _states.Add(GameStates.StartMenu, new GameStateStart(this));
+        }
```

Comment:
We're just measuring how long it takes to create an instance of `GameStateStart` and to add it to a list?